packages/components/src/components/tile/tile.scss

```
@@ -214,6 +231,12 @@ calcite-link {
   }
 }
 
+:host([selection-appearance="highlight"]) {
+  .container.selected {
+    background-color: var(--calcite-color-surface-highlight);
```

With the new public css token:

```suggestion
    background-color: var(--calcite-tile-background-color-highlight, var(--calcite-color-surface-highlight));
```